google-cloud-logging/src/test/java/com/google/cloud/logging/K8sContainerUtilTest.java

```diff
@@ -0,0 +1,33 @@
+package com.google.cloud.logging;
+
+import org.junit.Test;
+
+import static com.google.cloud.logging.K8sContainerUtil.getContainerNameFromHostName;
+import static com.google.common.truth.Truth.assertThat;
+
+public class K8sContainerUtilTest {
+
+  @Test
+  public void testGetContainerNameProperlyFormatted() {
+    assertThat(getContainerNameFromHostName("test-one-68449bf55d-44pdr"))
+        .isEqualTo("test-one");
+    assertThat(getContainerNameFromHostName("test-again-d667d8f5-rlxz5"))
+        .isEqualTo("test-again");
```

Comment:
is there a difference between these 2 usecases?

Reply:
Just different length in the deployment id.